client/src/pages/guide/english/java/arraylist/index.md

```diff
@@ -20,7 +20,11 @@ Since ArrayList implements *List*, an ArrayList can be created using the followi
   List<Integer> students = new ArrayList<>();
   ```
   
-  An ArrayList is dynamic, meaning it will grow in size if required and similarly shrink in size if elements are deleted from it. This is what makes it better to use than normal arrays. 
+  An ArrayList is dynamic, meaning it will grow in size if required and similarly shrink in size if elements are deleted from it. This is what makes it better to use than normal arrays.
+  
+  To clear ArrayList call clear() method;
+  variable.clear();
```

Comment:
It's missing the code tags like the rest of the text. The explicit example is not needed either.

To clear an ArrayList call its clear() method.